capture-server/tests/common.rs

```diff
@@ -39,6 +39,7 @@ pub static DEFAULT_CONFIG: Lazy<Config> = Lazy::new(|| Config {
         kafka_compression_codec: "none".to_string(),
         kafka_hosts: "kafka:9092".to_string(),
         kafka_topic: "events_plugin_ingestion".to_string(),
+        kafka_historical_topic: "events_plugin_ingestion_historical".to_string(),
```

Comment:
We'll need a better story around specifying 10 topics on several clusters, but that's for future us